segmentation/utils/spatial_analysis.py

```diff
@@ -53,202 +52,203 @@ def calculate_channel_spatial_enrichment(dist_matrices, marker_thresholds, all_d
 
     Args:
-        all_data: data including points, cell labels, and
+        all_fov_data: data including points, cell labels, and
             cell expression matrix for all markers
-        dist_mats: A dictionary that contains a cells x cells matrix with the euclidian
+        dist_matrices_dict: A dictionary that contains a cells x cells matrix with the euclidian
             distance between centers of corresponding cells for every fov
-        fovs: patient labels to include in analysis. If argument is none, default is all labels used
+        included_fovs: patient labels to include in analysis. If argument is none, default is all labels used
         bootstrap_num: number of permutations for bootstrap. Default is 1000
         dist_lim: cell proximity threshold. Default is 100
-
+        fov_col (string): column with the cell fovs (Default is SampleID)
+        cell_type_col (string): column with the cell types (Default is cell_type)
+        flowsom_col (string): column with the cell phenotype IDs (Default is FlowSOM_ID)
+        cell_label_col (string): column with the cell labels (Default is cellLabelInImage)
     Returns:
         values: a list with each element consisting of a tuple of
             closenum and closenumrand for each point included in the analysis
         stats: an Xarray with dimensions (points, stats, number of markers, number of markers) The included stats
             variables are:
             z, muhat, sigmahat, p, h, adj_p, and marker_titles for each point in the analysis"""
 
-    fov_col = "SampleID"
-    cell_type_col = "cell_type"
-    flowsom_col = "FlowSOM_ID"
-    cell_label_col = "cellLabelInImage"
-
     # Setup input and parameters
-    if fovs is None:
-        fovs = list(set(all_data[fov_col]))
-        num_fovs = len(fovs)
+    if included_fovs is None:
+        included_fovs = list(set(all_fov_data[fov_col]))
+        num_fovs = len(included_fovs)
     else:
-        num_fovs = len(fovs)
+        num_fovs = len(included_fovs)
 
     values = []
 
     # Error Checking
-    if not np.isin(fovs, all_data[fov_col]).all():
+    if not np.isin(included_fovs, all_fov_data[fov_col]).all():
         raise ValueError("Points were not found in Expression Matrix")
 
     # Extract the names of the cell phenotypes
-    pheno_titles = all_data[cell_type_col].drop_duplicates()
+    pheno_titles = all_fov_data[cell_type_col].drop_duplicates()
     # Extract the columns with the cell phenotype codes
-    pheno_codes = all_data[flowsom_col].drop_duplicates()
+    pheno_codes = all_fov_data[flowsom_col].drop_duplicates()
     # Get the total number of phenotypes
     pheno_num = len(pheno_codes)
 
     # Subset matrix to only include the columns with the patient label, cell label, and cell phenotype
-    fov_cluster_data = all_data[[fov_col, cell_label_col, flowsom_col]]
+    all_cluster_data = all_fov_data[[fov_col, cell_label_col, flowsom_col]]
```

Comment:
I like these changes, but I think it actually makes more sense to swap them. 

We should use cluster to refer to the cell_types, and pheno to refer to the matrix. 

So let's modify cell_type_col -> cluster_name_col, flowsom_col -> cluster_id_col, pheno_titles -> cluster_names, pheno_codes -> cluster_ids

all_cluster_data -> all_pheno_data